src/plugins/walletConnectToDapps/components/header/WalletConnectToDappsHeaderButton.tsx

```diff
@@ -0,0 +1,27 @@
+import { Button } from '@chakra-ui/react'
+import type { FC } from 'react'
+import { useState } from 'react'
+import { useTranslate } from 'react-polyglot'
+import { WalletConnectIcon } from 'components/Icons/WalletConnectIcon'
+
+import { WalletConnectModal } from '../modal/WalletConnectModal'
+
+export const WalletConnectToDappsHeaderButton: FC = () => {
+  const [isOpen, setOpen] = useState(false)
+  const translate = useTranslate()
+  return (
+    <>
+      <Button
+        leftIcon={<WalletConnectIcon />}
+        width='full'
+        onClick={() => setOpen(true)}
```

Comment:
Nitipck, but if using a toggle, we won't have to add an anonymous function here, making it easier on stack traces and new Function Object references